File: Java/level 4/Representation of numbers.java
public class Expressions {

	public int expressions(int num) {
		int answer = 1;
		int tem = 0;
  	for(int a=1; a<=num/2; a++){
  		tem = a;
    	for(int b=a+1; b<=num/2+1; b++){
      	tem+=b;
    		if(tem==num){
        	answer++;
      		break;
      	}
    	}
  	}
	return answer;
	}

	public static void main(String args[]) {
		Expressions expressions = new Expressions();
		// 아래는 테스트로 출력해 보기 위한 코드입니다.
		System.out.println(expressions.expressions(15));
	}
}

File: Javascript/level 4/Representation of numbers.js
function expressions(num) {
    var answer = 1;
    var tem = 0;
  for(var a=1; a<=num/2; a++){
  console.log("A",a)
      tem = a;
    for(var b=a+1; b<=num/2+1; b++){
    console.log("B",b)
      tem+=b;
        if(tem==num){
        answer++;
          break;
      }
    }
  }
return answer;
}

// 아래는 테스트로 출력해 보기 위한 코드입니다.
console.log(expressions(15));